.github/workflows/story_block.yml

```
@@ -0,0 +1,32 @@
+name: StoryBlock CI
```

Now that it is a two word feature name:
```suggestion
name: Story Block CI
```